recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: cobra
+  version: "0.4.1"
+
+source:
+  fn: cobra-0.4.1.tar.gz
+  url: https://pypi.python.org/packages/source/c/cobra/cobra-0.4.1.tar.gz
+  md5: a96799117abed427333bb57e007a8b82
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - glpk
+
+  run:
+    - python
+    - six
+    - numpy
+    - scipy
+    - lxml
+    #- python-libsbml
```

Comment:
Alright, maybe we can just include Mac and Linux builds if you have something. Though it can totally be a separate PR.

I haven't used this so wasn't sure what went into getting it built. I was hoping `python-*` meant easy. :wink: